### PR TITLE
Add date format to exported logs

### PR DESCRIPTION
**Pull request by KCeh**
## :camera: Screenshots
Screenshot_20240707_163138

## :page_facing_up: Context
As requested in https://github.com/infinum/android-sentinel/issues/47 exported logs now have human-readable timestamp (same format as logcat). Also, to have the consistency I added same date-time format to the list of logs (and log files)

## :pencil: Changes
Added date-time formating when writing logs to files. Changed date-time formatting in list of logs 

## :hammer_and_wrench: How to test
Try out sample app

### CODE REVIEW

File: tool-timber/src/main/kotlin/com/infinum/sentinel/SentinelFileTree.kt [hunk 1]
```diff
@@ -21,6 +22,8 @@ internal class SentinelFileTree(
     private val logFileResolver = LogFileResolver(context)
 
     override fun log(priority: Int, tag: String?, message: String, t: Throwable?) {
+        val dateTimeFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss. SSSZ", Locale.getDefault())
```
Comment (JonatanPlesko) [inline tool-timber/src/main/kotlin/com/infinum/sentinel/SentinelFileTree.kt:25]: If I'm interpreting the screenshot correctly, this space seems unnecessary.

```suggestion
        val dateTimeFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ", Locale.getDefault())
```

File: tool-timber/src/main/kotlin/com/infinum/sentinel/ui/logger/LoggerAdapter.kt [hunk 2]
```diff
@@ -5,12 +5,16 @@ import android.view.ViewGroup
...
 import com.infinum.sentinel.databinding.SentinelItemLogBinding
+import java.text.SimpleDateFormat
+import java.util.Locale
 
 internal class LoggerAdapter(
     private val onListChanged: (Boolean) -> Unit,
     private val onClick: (SentinelFileTree.Entry) -> Unit
 ) : ListAdapter<SentinelFileTree.Entry, LoggerViewHolder>(LoggerDiffUtil()) {
 
+    private val dateTimeFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss. SSSZ", Locale.getDefault())
```
Comment (JonatanPlesko) [inline tool-timber/src/main/kotlin/com/infinum/sentinel/ui/logger/LoggerAdapter.kt:16]: Here too, and it also makes me wonder if we could just share this pattern or formatted between the two files 😄 

```suggestion
    private val dateTimeFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ", Locale.getDefault())
```

Comment (KCeh) [reply tool-timber/src/main/kotlin/com/infinum/sentinel/ui/logger/LoggerAdapter.kt:16]: Yea, of course

**Review by JonatanPlesko — APPROVED** [review]
Changes look good to me, but best if someone with more knowledge of the library code looks into it as well. Nice to see this improvement! 👏

**Pull request merged**